core/libdivecomputer.c

```diff
@@ -1318,6 +1318,10 @@ dc_status_t divecomputer_device_open(device_data_t *data)
 #ifdef SERIAL_FTDI
 		if (!strcasecmp(data->devname, "ftdi"))
 			return ftdi_open(&data->iostream, context);
+#endif
+#ifdef __ANDROID__
+		if (!strcasecmp(data->devname, "usb-serial-for-android"))
```

Comment:
do we really want such a long device name? This is only used on Android, so why have "-for-android" be part of the name? `"usb-serial"` seems sufficient...

Reply:
agree

Reply:
fixed